certbot/cli.py

```diff
@@ -440,8 +440,8 @@ def __init__(self, args, plugins, detect_defaults=False):
         }
 
         # List of topics for which additional help can be provided
-        HELP_TOPICS = ["all", "security", "paths", "automation", "testing"] + list(self.VERBS)
-        HELP_TOPICS += self.COMMANDS_TOPICS + ["manage"]
+        HELP_TOPICS = ["all", "security", "paths", "automation", "testing"]
+        HELP_TOPICS += list(self.VERBS) + self.COMMANDS_TOPICS + ["manage"]
```

Comment:
This was done in the PR by another author this PR is based on in [this commit](https://github.com/certbot/certbot/pull/4195/commits/93e814a85fb82e1cd201ce90d69828d89e52371d) because a new help topic was added in a prior commit (and later removed).

As an extremely minor positive side effect though, the line lengths are here now follow our [style](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) [guides](https://google.github.io/styleguide/pyguide.html#Line_length).